Write instruction how to add case study (#454)
kyma-project/website: docs/add-adopter.md

> # Add adopter

Reviewer: ```suggestion
# Add a new adopter
```

> Follow these steps to add a new adopter of Kyma on the website:

Reviewer: ```suggestion
Follow these steps to add information on a new Kyma adopter to the **Used by** section on the website's landing page:
```

1. Fork repository and create new branch in your forked repo.

> 2. Update the `adopters.yaml` file in `content/adopters` folder with new list item with information about your company. See under the list for more information.

Reviewer: ```suggestion
2. Update the `adopters.yaml` file in the `content/adopters` folder with a new list item with your company's details. 
```

Reviewer: I would give this entire fragment here as it refers to point 2 in particular: `Each element of the list in the `adopters.yaml` file should have the following format:...`

3. Make a pull request.
4. Wait for the review.

> 5. After the merge, the site rebuilds automatically and the your company appears on `Used by` in landing page of our website.

Reviewer: ```suggestion
5. After the merge, the site rebuilds automatically and your company appears in the **Used by** section on the landing page.
```

> 5. After the merge, the site rebuilds automatically and the your company appears on `Used by` in landing page of our website.

Reviewer: ```suggestion
5. After the merge, the site rebuilds automatically and your company's case study appears in the **Used by** section on the landing page.
```

> The each element of list in `adopters.yaml` file should have the following format:

Reviewer: ```suggestion
Each element of the list in the `adopters.yaml` file should have the following format:
```

``` yaml
- websiteUrl: {WEBSITE_URL}
  url: {URL}
  logo: {LOGO}
  content: {CONTENT}
```

Replace these parameters with real values:

> - `{WEBSITE_URL}` is an url to your company website.

Reviewer: ```suggestion
- `{WEBSITE_URL}` is a URL to your company's website.
```

> - `{URL}` is an url to case study written on your company or our blog.

Reviewer: ```suggestion
- `{URL}` is a URL to your case study published either on your company's website or our blog.
```

> - `{LOGO}` is a path to logo of your company. It can be as relative path to image provided with PR in `content/adopters/logos` folder or path to external source.

Reviewer: ```suggestion
- `{LOGO}` is a path to your company's logo. It can be either a relative path to an image added under the `content/adopters/logos` folder in your PR or a path to an external source.
```

> - `{CONTENT}` is a short brief about feelings from adopting Kyma in your business. Only pure text and maximum 140 characters.

Reviewer: ```suggestion
- `{CONTENT}` is a short brief about feelings from adopting Kyma in your business. You can only add text and a maximum of 140 characters.
```

> > **NOTE**: If you have a blog post in our blog, then `{URL}` should be as a relative link to appropriate blog post with `/blog` prefix, for example `/blog/2018/7/24/introduction-project-kyma/`.

Reviewer: ```suggestion
> **NOTE:** If your case study is a post on our blog, then `{URL}` should be a relative link to the appropriate post with the `/blog` prefix. For example, add `/blog/2018/7/24/introduction-project-kyma/`.
```

## Examples

> 

Reviewer: `See these examples for reference:`
